cql-unit-tests/README.md

```diff
@@ -0,0 +1,25 @@
+## CQL Unit Tests
+
+The CQL Unit Tests are included in this repo as a companion JavaScript (TypeScript) module to the Covid-Ed-Ig CQL repository.
+These tests internally use the CQL-Exec-Fhir and CQL-Execution packages to run the CQL via the JavaScript engine.
+
+
+### Pre-requisites 
+Preferably install both a java version manager and a ruby version manager.
+Install the following:
+- Java 1.8
+- Ruby 3.0.0 
+- Jekyll (https://jekyllrb.com/docs/)
+
+### Running Unit Tests
+
+To run the unit tests with the latest CQL code, you need to have the above dependencies installed on your machine.
+
+1. In the covid-ed-ig directory run `./_genonce.sh` script.
+   - If you get an 'ig publisher not found' message, run `bash _updatePublisher.sh --force --yes --skip-ping` script.
+     Then run `./_genonce.sh` script to generate the ELM (json) of the CQL. The output will be located in the `/output` directory
+2. Run `node localizeLibraryPaths.js [PATH TO CQL LIBRARY JSON]` (for example: `/output/Library-COVID19EmergencyDeptAssessment-2.json`) for each library being used in tests
```

Comment:
Would it be possible to optimize this into a script so we can just run a script that will do the `node localizeLibraryPath.js` for all the relevant files? Otherwise, every time we rerun the genonce script, we have to run this several times to do every file.

Reply:
I guess I'm also wondering what it means to update the path values for the libraries. Could you give an example of what this is doing?

Reply:
Yeah the plan is to hopefully optimize the script too since it takes a while to run _genonce.